### DISCUSSION TITLE
Remove fork relationship from original repo?

### DISCUSSION THREAD

**Discussion by philmtd** [helmfile/helmfile#191] [Questions] [answered]
Hey, I just thought whether it could be useful to remove the official fork relationship from [roboll/helmfile](https://github.com/roboll/helmfile)?

As this is the new official home of this project it would probably be a good message for everyone out there using it who are uncomfortable or confused by the fork relationship. Additionally I saw in #154 that code search is not available in forked repos.

Is this an interesting idea? Or am I missing a reason for this repository to remain a fork?

Forks can be detached by consulting this page, by the way: https://support.github.com/request/fork

**Comment by philmtd** [comment]
Oh and I just found #116 which is party identical to this discussion. Looks like some of us are interested in this. :)

**Comment by yxxhero** [comment]
@mumoshu What do you think of this idea? Thanks very much.

**Comment by mumoshu** [comment, marked as answer]
I wasn't even aware that Code Search isn't available for forks! And yeah, removing the fork relationship would make this repo look more like the upstream. Makes sense a lot.

I just headed over to https://support.github.com/request/fork you shared. It started a session with the virtual assistance so I provided everything needed to detach this repo from roboll/helmfile. Let's hold on for a few days so that hopefully the request will get processed!

**Reply by philmtd** [reply]
Thanks! Let‘s see how fast GitHub will check this.

**Reply by philomory** [reply]
Looks like it's done, some time in the past hour or so.

**Reply by philmtd** [reply]
Yes, awesome!

**Reply by yxxhero** [reply]
very good. It's a very good news.
